### PR TITLE
Support substitution of vscode variables in ctestPath like in cmakePath

### PR DESCRIPTION
**Pull request by FakeTruth**
### This changes ctestPath behavior

The following changes are proposed:

- Substitute VSCode variables for ctestPath in the same way as for cmakePath.

## The purpose of this change

Enables users to use variables like `${workspaceFolder}` in their ctestPath. cmakePath already allows this.

### CODE REVIEW

**Review by elahehrashedi — CHANGES_REQUESTED** [review]

File: src/paths.ts [hunk 1]
```diff
@@ -241,6 +220,30 @@ class Paths {
     return raw;
   }
 
+  async expandStringWithVSCodeVars(tmpl: string, wsc: DirectoryContext): Promise<string> {
```
[the remark this answers is not in the document]
Comment (FakeTruth) [reply src/paths.ts]: Thanks for the review, I've updated the pull request

**Review by elahehrashedi — APPROVED** [review]

File: src/paths.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (elahehrashedi) [reply src/paths.ts]: @FakeTruth Thank you for your prompt response. I approved your PR. Please merge your PR by clicking on "Merge Pull Request" green button in below.

[the remark this answers is not in the document]
Comment (FakeTruth) [reply src/paths.ts]: Thanks approval, unfortunately there's no merge button for me. It says:
`Only those with write access to this repository can merge pull requests.`

[the remark this answers is not in the document]
Comment (bobbrow) [reply src/paths.ts]: I merged it.

**Pull request merged**